draft-ietf-oauth-status-list.md

```diff
@@ -462,6 +464,50 @@ If caching is required (e.g., to enable the use of alternative mechanisms for ho
 
 TBD
 
+# Batch Fetching {#batch-fetching}
+
+To allow a Relying Party to fetch all status lists for a specific type of credential or a specific issuer, an optional mechanism is provided to retrieve an Object containing a list of Status Lists called Status List Aggregation. This mechanism is intended to optimize fetching and caching mechanisms and allow offline validation of the status.
+
+There are two options that a Relying Party can use to discover the relevant Status List Aggregation URI.
+An issuer MAY support any of these mechanisms:
+
+- Issuer metadata: The issuer publishes an URI which links to Status List Aggregation
+- Status List Parameter: The issuer can choose to add an additional claim to the Status List (Token) that links to the Status List Aggregation.
+
```

Comment:
maybe turn these chapters around? Now that we mentioned the two mechanisms, giving the details on those fits better